lldb/test/API/commands/statistics/basic/main.cpp

```diff
@@ -1,7 +1,13 @@
 // Test that the lldb command `statistics` works.
+#include <string>
+
+void foo() {
+  std::string str = "hello world";
```

Comment:
instead of relying on `std::string` formatter should we add a type summary for a dummy type? And test against that? Feels a bit more hermetic